ruslingdk/app/views/partials/_event_accordion.html.erb

```diff
@@ -0,0 +1,56 @@
+<h5 style="text-align: center">
+  <% if condition == 0 %>
+    <%= t("events.today") %>
+  <% elsif condition == 1 %>
+    <%= t("events.upcoming") %>
+  <% else %>
+    <%= t("events.previous") %>
+  <% end %>
+</h5>
+
+<div id="<%= accordionID %>">
+    <% @domain.events.order(:begin_at).each do |event| %>
+      <% if Date.parse(event["begin_at"].to_s).compare_without_coercion(Date.today) == condition %>
```

Comment:
Maybe querybuild like `@domain.events.order(:begin_at).today` with scopes on `Event`-model `today`, `future`, `past` scopings

Reply:
send som local med i viewet, og brug `.send(condition.in?([:today, :future, :past]) ? condition : :all)`

Reply:
(for validation)